### ISSUE TITLE
iPhone connected, not jailbroken

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MziRintu/iphonebrowser#27]
```
What steps will reproduce the problem?
1. Restore iPhone
2. Jailbreak with ziphone, activate with itune
3. turn on iphonebrowser

What is the expected output? What do you see instead?
expect to browse iPhone director, instead I see iPhone connected, not
jailbroken

What version of the product are you using? On what operating system?
1.1.4 Windows Xp



```

Original issue reported on code.google.com by `gwan...@gmail.com` on 8 Mar 2008 at 5:05

**Comment by GoogleCodeExporter** [comment]
```
you can fixe this issue by add source i.unlock.no 
then install "ZiPhone iBrickr Fix" from Unlocking Tools
```

Original comment by `sategr...@gmail.com` on 15 Apr 2008 at 4:21

**Comment by GoogleCodeExporter** [comment]
```
THIS WILL GET RESOLVED LIKE THIS:

install ziphone ibrikr fix patch for 1.1.4 even if you dont have ibrickr.

Add i.unlock.no as a source and install that patch.

ENJoY!!
```

Original comment by `vikramsh...@gmail.com` on 2 Jun 2008 at 8:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `NetMage....@gmail.com` on 11 Sep 2008 at 6:30
- Changed state: **Invalid**
